a8cteam51/github-repo-init
# Adds file tree scaffolding for themes and plugins.

#### Changes proposed in this Pull Request

When a new project repo is created, will add initial template files for a theme and/or plugin, based on prompt input. These files can be replaced. They are mainly to demonstrate the project's structure.

#### Testing instructions



* Open up your Terminal application, clone this repository to your local machine, and move into the cloned `github-repo-init` directory: `git clone git@github.com:a8cteam51/github-repo-init.git && cd github-repo-init`.
* Run the init script: `sh github-repo-init.sh`.
* Follow the prompts from your Terminal - be sure to enter yes to create a theme and/or plugin.
* Once the script completes, you can view your new repository on GitHub via the link provided.
* At your repository on GitHub, visit **Settings > Branches** and confirm that theme or plugin files have been created under `themes/theme_name` or `plugins/plugin_name`, respectively.
